generators/app/__snapshots__/generator.spec.mts.snap

```diff
@@ -703,9 +703,9 @@ exports[`generator - app with gateway should match snapshot 1`] = `
   "devDatabaseUsername": "jhipster",
   "devHibernateDialect": "org.hibernate.dialect.H2Dialect",
   "devJdbcDriver": "org.h2.Driver",
-  "devJdbcUrl": "jdbc:h2:file:./target/h2db/db/jhipster;DB_CLOSE_DELAY=-1",
+  "devJdbcUrl": "jdbc:h2:file:./target/h2db/db/jhipster;DB_CLOSE_DELAY=-1;TIME ZONE=UTC",
   "devLiquibaseUrl": "jdbc:h2:file:\${project.build.directory}/h2db/db/jhipster",
-  "devR2dbcUrl": "r2dbc:h2:file:///./target/h2db/db/jhipster;DB_CLOSE_DELAY=-1",
+  "devR2dbcUrl": "r2dbc:h2:file:///./target/h2db/db/jhipster;DB_CLOSE_DELAY=-1;TIME ZONE=UTC",
```

Comment:
I found there an `java.net.URISyntaxException` in r2dbc, it wants the space to be escaped probably:
```
java.net.URISyntaxException: Illegal character in path at index 74: r2dbc:///./build/h2db/db/timetst_gateway;DB_CLOSE_DELAY=-1;MODE=MYSQL;TIME ZONE=UTC
	at java.base/java.net.URI$Parser.fail(URI.java:2973)
	at java.base/java.net.URI$Parser.checkChars(URI.java:3144)
	at java.base/java.net.URI$Parser.parseHierarchical(URI.java:3226)
	at java.base/java.net.URI$Parser.parse(URI.java:3174)
	at java.base/java.net.URI.<init>(URI.java:623)
	at java.base/java.net.URI.create(URI.java:904)
```
But if we URL-encode the empty space in `TIME ZONE` -> `TIME%20ZONE`, then it goes away.

Reply:
Thanks @shrralis, I'm just not sure if we should keep this time zone setting or just update the test to pass without it, so waiting for a response from @DanielFran, @mshima or maybe @mraible?